### PR TITLE
#12442 Tensorflow math softmax implementation

### PR DESCRIPTION
**Pull request by veyselsapan**

### CODE REVIEW

**Comment by aditya-hari** [comment]
Hey! Thanks for the contribution. Can you please link this to the issue that this solves? You can find the relevant instructions in the [contributing guide](https://lets-unify.ai/ivy/overview/contributing.html)

**Comment by veyselsapan** [comment]
Close [#12442](https://github.com/unifyai/ivy/issues/12442)

**Comment by veyselsapan** [comment]
Hi @YushaArif99,
I fixed and it passed all run test but I have no idea about lint / check-formatting,
Could you please help ?

**Review by YushaArif99 — APPROVED** [review]
Hi @veyselsapan 

Thanks for fixing the tests! You don't need to worry about the lint formatting errors as they are unrelated to your changes. I think the PR looks in a good state. I will now be merging this PR. 

Thanks for the contribution! 🙂
